lib/Arch/AArch32/Decode.cpp

```diff
@@ -999,7 +999,8 @@ static bool DecodeCondition(Instruction &inst, uint32_t cond) {
 }
 
 std::optional<uint64_t> EvalReg(const Instruction &inst,
-                                const Operand::Register &op) {
+                                const Operand::Register &op, bool &is_linkreg) {
+  is_linkreg = (op.name == kIntRegName[kLRRegNum] || op.name == "LR");
```

Comment:
This needs to be saturating:
```c++
if (!is_linkreg) {
  is_linkreg = (op.name == kIntRegName[kLRRegNum] || op.name == "LR");
}
```

Reply:
Also rename to `uses_linkreg`, as `uses` communicates saturation, whereas `is` does not.